<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>258</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>290</width>
    <height>258</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>290</width>
    <height>258</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>272</width>
        <height>199</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>272</width>
        <height>199</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QGrouoBox{
	background-color: #000000;
}</string>
      </property>
      <property name="title">
       <string>Calculator</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="0">
          <widget class="QPushButton" name="Button_4">
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color : #87CEEB;
border: 1px solid gray;
padding: 5px;
}

QPushButton: pressed{
background-color : #A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
           </property>
           <property name="text">
            <string>4</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="Button_7">
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color : #87CEEB;
border: 1px solid gray;
padding: 5px;
}

QPushButton: pressed{
background-color : #A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
           </property>
           <property name="text">
            <string>7</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QPushButton" name="Button_clear">
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color : #FFA500;
border: 1px solid gray;
padding: 5px;
}

QPushButton: pressed{
background-color : #A9A9A9;
border: 1px solid gray;
padding: 5px;
}
</string>
           </property>
           <property name="text">
            <string>C</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QPushButton" name="Button_plus">
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color : #00FF7F;
border: 1px solid gray;
padding: 5px;
}

QPushButton: pressed{
background-color : #A9A9A9;
border: 1px solid gray;
padding: 5px;
}
</string>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item row="3" column="4">
          <widget class="QPushButton" name="Button_sqrt">
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color : #00FF7F;
border: 1px solid gray;
padding: 5px;
}

QPushButton: pressed{
background-color : #A9A9A9;
border: 1px solid gray;
padding: 5px;
}
</string>
           </property>
           <property name="text">
            <string>√</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="Button_0">
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color : #87CEEB;
border: 1px solid gray;
padding: 5px;
}

QPushButton: pressed{
background-color : #A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="Button_6">
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color : #87CEEB;
border: 1px solid gray;
padding: 5px;
}

QPushButton: pressed{
background-color : #A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
           </property>
           <property name="text">
            <string>6</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="Button_5">
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color : #87CEEB;
border: 1px solid gray;
padding: 5px;
}

QPushButton: pressed{
background-color : #A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
           </property>
           <property name="text">
            <string>5</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="Button_2">
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color : #87CEEB;
border: 1px solid gray;
padding: 5px;
}

QPushButton: pressed{
background-color : #A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
           </property>
           <property name="text">
            <string>2</string>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QPushButton" name="Button_procent">
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color : #00FF7F;
border: 1px solid gray;
padding: 5px;
}

QPushButton: pressed{
background-color : #A9A9A9;
border: 1px solid gray;
padding: 5px;
}
</string>
           </property>
           <property name="text">
            <string>%</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QPushButton" name="Button_plusminus">
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color : #00FF7F;
border: 1px solid gray;
padding: 5px;
}

QPushButton: pressed{
background-color : #A9A9A9;
border: 1px solid gray;
padding: 5px;
}
</string>
           </property>
           <property name="text">
            <string>±</string>
           </property>
          </widget>
         </item>
         <item row="4" column="4">
          <widget class="QPushButton" name="Button_equals">
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color : #00FF7F;
border: 1px solid gray;
padding: 5px;
}

QPushButton: pressed{
background-color : #A9A9A9;
border: 1px solid gray;
padding: 5px;
}
</string>
           </property>
           <property name="text">
            <string>=</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="Button_8">
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color : #87CEEB;
border: 1px solid gray;
padding: 5px;
}

QPushButton: pressed{
background-color : #A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
           </property>
           <property name="text">
            <string>8</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QPushButton" name="Button_multiply">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color : #00FF7F;
border: 1px solid gray;
padding: 5px;
}

QPushButton: pressed{
background-color : #A9A9A9;
border: 1px solid gray;
padding: 5px;
}
</string>
           </property>
           <property name="text">
            <string>×</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="Button_dot">
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color : #87CEEB;
border: 1px solid gray;
padding: 5px;
}

QPushButton: pressed{
background-color : #A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
           </property>
           <property name="text">
            <string>.</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="Button_1">
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color : #87CEEB;
border: 1px solid gray;
padding: 5px;
}

QPushButton: pressed{
background-color : #A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
           </property>
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QPushButton" name="Button_3">
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color : #87CEEB;
border: 1px solid gray;
padding: 5px;
}

QPushButton: pressed{
background-color : #A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
           </property>
           <property name="text">
            <string>3</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QPushButton" name="Button_division">
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color : #00FF7F;
border: 1px solid gray;
padding: 5px;
}

QPushButton: pressed{
background-color : #A9A9A9;
border: 1px solid gray;
padding: 5px;
}
</string>
           </property>
           <property name="text">
            <string>÷</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="Button_9">
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color : #87CEEB;
border: 1px solid gray;
padding: 5px;
}

QPushButton: pressed{
background-color : #A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
           </property>
           <property name="text">
            <string>9</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QPushButton" name="Button_minus">
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color : #00FF7F;
border: 1px solid gray;
padding: 5px;
}

QPushButton: pressed{
background-color : #A9A9A9;
border: 1px solid gray;
padding: 5px;
}
</string>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="5">
          <widget class="QLineEdit" name="Display">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>290</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
